codecov.yml

```diff
@@ -1,15 +1,40 @@
+# See documentation at https://docs.codecov.io/docs/codecovyml-reference
+
+codecov:
+  notify:
+    # Wait for Jest & PHPUnit reports
+    after_n_builds: 2
+
 coverage:
   status:
-    patch: false
-    changes: false
+    patch:
+      default:
+        base: auto
+        only_pulls: true
+        target: auto
+        threshold: 50%
+
     project:
       default:
         target: auto
         threshold: 0.1%
+      php:
+        paths:
+          - includes
+          - src
+      js:
+        paths:
+          - assets/src
 
-comment: off
+comment:
+  layout: 'reach, diff, flags, files'
```

Comment:
Are the flags currently serving an actual purpose.

In the main view, this only shows a warning right now:
![Image 2020-09-16 at 12 45 29 PM](https://user-images.githubusercontent.com/83631/93327381-85738500-f81a-11ea-8e47-2ddb1620dcd7.png)

Reply:
Yes they aren't shown in the main view right now, but a summary is shown in the [CodeCov report comment](https://github.com/ampproject/amp-wp/pull/5341#issuecomment-687931009):

![image](https://user-images.githubusercontent.com/16200219/93688483-b4595780-fab5-11ea-9cac-31258323660f.png)

and when viewing a file via the CodeCov interface (eg. [assets/src/utils/use-window-width.js](https://codecov.io/gh/ampproject/amp-wp/pull/5341/src/assets/src/utils/use-window-width.js):

![image](https://user-images.githubusercontent.com/16200219/93688560-33e72680-fab6-11ea-90ec-60c2740c5def.png)